### DISCUSSION TITLE
vuepress2.0.0-rc.9用不了评论插件了

### DISCUSSION THREAD

**Discussion by wankaiyi** [vuepress-theme-hope/vuepress-theme-hope#4070] [Q&A] [answered]
因为vuepress2.0.0-rc.9以前的版本存在一些问题，所以升级到了这个版本，在下载依赖的时候发现评论插件用不了了，刚刚看了一下tag，发现评论插件在vuepress-theme-hope2.0.0-rc.31被删除了，后序还会加回来吗
![image](https://github.com/vuepress-theme-hope/vuepress-theme-hope/assets/129262385/c1c72138-41b8-4687-98ff-e94815dd40fd)

**Comment by Mister-Hope** [comment, marked as answer]
使用 @vuepress/plugin-comment
